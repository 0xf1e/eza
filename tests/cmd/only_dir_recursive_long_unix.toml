bin.name = "eza"
args = "tests/itest --only-dirs --recurse --long --no-time --no-user --no-file-size"
status.code = 3
