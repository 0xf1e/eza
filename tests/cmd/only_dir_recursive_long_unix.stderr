eza: Unknown argument --no-file-size
